### ISSUE TITLE
Remember original location after clicking on settings and return to it when clicking on the logo

### ISSUE THREAD

**Issue by Bauke** [kickscondor/fraidycat#27]
Say you're scrolled down and deep into a specific tag of yours, and for whatever reason you want to change a setting (there's not much to change yet but in the future maybe). Right now when you click on the fraidycat logo to go back to the feeds, it takes you to the index/home tag. I think it would be nice if instead of going to the index, it would take you back to where you were in the tag.

**Comment by kickscondor** [comment]
If you hit the back button, you should be back to the same page.

My concern with doing this to the fraidycat logo is that I personally expect it to go back to home. But I can see what you're saying... I appreciate this perspective. I will label this 'considering'.

**Comment by jmacdotorg** [comment]
I actually arrived at this issue with "Please add a back button" in mind — and I suspect that this is because I'm a weirdo running Fraidycat as a stand-alone app (in Mac) instead of as a browser extension.

Adding a separate back button to the **Edit a Follow** screen would solve both my issue and the OP's, but I don't know if you want to introduce that level of UI redundancy.

**Comment by kickscondor** [comment]
Yes for sure - the Electron app is only a few days old, so I still have a pile of changes to make and this is among them. I plan to add some basic navigation (much like there is a status bar in Electron as well.) Thank you for the reminder, jmac!
